share/completions/apk.fish

```diff
@@ -20,18 +20,18 @@ complete -f -c apk -l force-overwrite -d "Overwrite files in other packages"
 complete -f -c apk -l force-refresh -d "Don't use cached files"
 complete -f -c apk -s U -l update-cache -d "Alias for --cache-max-age 1"
 complete -f -c apk -l progress -d "Show a progress bar"
-complete -f -c apk -l progress-fd -d "Write progress to FD"
+complete -F -c apk -l progress-fd -r -d "Write progress to FD"
```

Comment:
this should probably stay without file completions; FD = file descriptor, so 1 for stdout, 2 for stderr, etc.